src/TournamentManager.ts

```diff
@@ -358,10 +330,7 @@ export class TournamentManager implements TournamentInterface {
 			return;
 		}
 		// allow confirmed user to resubmit
-		const allTourns = await this.database.listTournaments();
-		const confirmedTourns = allTourns.filter(
-			t => t.players.includes(msg.author) && t.status === TournamentStatus.PREPARING
-		);
+		const confirmedTourns = await this.database.getConfirmedTournaments(msg.author);
```

Comment:
does this no longer filter for preparing? a player being able to change their list while in progress is no bueno

Reply:
It's all in that new function now!